specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -152,7 +153,24 @@ Consider at least the below topics but you do not need to cover those that are n
       * Is it logged?
   * What secrets are needed by the components? How are these secrets secured and attained?
 -->
-
+### CICD
+
+#### e2e test image
+To run e2e tests alot of dependencies are needed (browsers, build systems etc...).
+Cypress(e2e framework) are actually aware of this and provide a base image.
+We extend this with extra functionality we need.
+A docker image is built with all these dependencies to make e2e testing easier in gitlab.
+
+it contains:
+1. Angluar (build)
+2. Chrome Browser (for UI testing)
+3. Command cli utils (curl, git, zip etc...) (auxiliary functions/packaging reports etc..)
+4. Sonar (Code Quality checking)
+5. Npm and Nvm (build)
+
```

Comment:
I am not sure, are the listed entries preinstalled in the official Cypress docker image, and what do we need to install in addition, and why?

Reply:
Only the browser is installed in the Cypress image.
 we install everything else. 
 Again like the previous comment I feel that the dockerfile is self documenting in that it has comments about what it is installing and ppl can read details about the base image if they like?